packages/gst_plugins_base.rb

```diff
@@ -0,0 +1,51 @@
+require 'package'
+
+class Gst_plugins_base < Package
+  description 'An essential, exemplary set of elements for GStreamer'
+  homepage 'https://gstreamer.freedesktop.org/modules/gst-plugins-base.html'
+  version '1.14.4'
+  source_url 'https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.14.4.tar.xz'
+  source_sha256 'ca6139490e48863e7706d870ff4e8ac9f417b56f3b9e4b3ce490c13b09a77461'
+
+  binary_url ({
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gst_plugins_base-1.14.4-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+     x86_64: 'f858e5ccb578bcff6db1cdd143727b8feec5cf153d0b438af4925827e6fabcc1',
+  })
+
+  depends_on 'gstreamer'
+  depends_on 'mesa'
```

Comment:
No need to include mesa since it's already in libglu. :)

Reply:
Wow. At this point, this is starting to sound a little absurd.

Reply:
Well, if it prevents dependency loops then it doesn't sound so absurd after all, right?

Reply:
Yeah. It wouldn't be a dependency loop, just a redundant dependency. Only maybe a couple of seconds shaved off the time of install.

Reply:
Yep, all this adds up especially since you have so many dependencies in this package already.

Reply:
Maybe. It would be nice to have Chromebrew consolidate redundant dependencies by itself.

Reply:
You are welcome to figure out that algorithm and submit a PR.

Reply:
No. I have no knowledge in Ruby.